hyppo/independence/hhg.py

```diff
@@ -95,17 +108,52 @@ class HHG(IndependenceTest):
         \mathrm{HHG}_n (x, y) = \sum_{i=1}^n \sum_{j=1, j \neq i}^n S(i, j)
 
     The p-value returned is calculated using a permutation test using
-    :meth:`hyppo.tools.perm_test`.
+    :math:`hyppo.tools.perm_test`.
+
+    For the fast version of the test, the test statistic is derived as follows:
+
+    Let :math:`x` and :math:`y` be :math:`(n, p)` samples of random variables
+    :math:`X` and :math:`Y`. A center point - the center of mass of points in 'X' and 'Y'
+    - is chosen. For every sample :math:`i`, calculate the distances from the center point
+    in :math:`x` and :math:`y` and denote this as :math:`d_x(x_i)`
+    and :math:`d_y(y_i)`. This will create a 1D collection of distances for each
+    sample group.
+
+    From these distances, we can calculate the Hoeffding's dependence score between
+    the two groups - denoted as D - using,
+
+    .. math::
+
+        D = \frac{(n-2) (n-3) D_{1} + D_{2} - 2(n-2) D_{3}}
```

Comment:
Would use the `align` LaTeX environment here

Reply:
Added alignment.